app/views/panels/edit.html.haml

```diff
@@ -1,7 +1,23 @@
 - content_for(:page_title, "Edit Panel - #{@grant.name}")
 
-%h1
-  = @grant.name
+%h1#grant-name
```

Comment:
I am not sure if this needs to be a link with the breadcrumbs below. I am not sure what is more usable. @lmkw ?